ddl/ddl_api.go

```diff
@@ -4778,6 +4781,93 @@ func (d *ddl) DropIndex(ctx sessionctx.Context, ti ast.Ident, indexName model.CI
 	return errors.Trace(err)
 }
 
+func (d *ddl) DropIndexes(ctx sessionctx.Context, ti ast.Ident, specs []*ast.AlterTableSpec) error {
```

Comment:
This function is quite similar to `DropIndex`, can we combine them together and use `DropIndexes` for dropping single index as well?

Reply:
👌